### ISSUE TITLE
More granular control over project privacy

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#1006]
We've been moving to a more simplified, more defaulted-to-open data privacy model, which is great all round.

But there are some situations where the blanket solution doesn't work. Over the past few months we've had a couple questions regarding privacy at particular stations. This is where a project has open data settings (and fully supports that) but there is just _one_ station that is sensitive. The recent examples have been bat hibernacula.

I don't believe we have a great way of dealing with these, other than putting the target stations into a separate project and changing the access level.

Even then, with the simplified approach, do we even have the option of defining access at a station level vs project wide?

**Comment by cjardine-bsc** [comment]
nope. We do not have a way to define privacy for a specific station, other than moving it into it's own project.

**Comment by StuMackenzie** [comment]
Is that how we've been handling the private stations to date? @Hussell
